assets/js/hooks/cell.js

```diff
@@ -198,6 +198,8 @@ const Cell = {
         if (this.isFocused && this.insertMode) {
           this.currentEditor().focus();
         }
+
+        this.sendCursorHistory();
```

Comment:
I don't think we need to send it here, this handler only refocuses the editor if the focus escapes temporarily.